sphinx/builders/epub.py

```diff
@@ -187,7 +187,8 @@ class EpubBuilder(StandaloneHTMLBuilder):
     add_permalinks = False
     # don't add sidebar etc.
     embedded = True
-
+    # dont' create links to original images from images
+    override_post_process_images = False
```

Comment:
Oh, I missed the configuration name. It is already named as `html_scaled_image_link`.
It would be nice to use related name.